internal/packetfabric/flex_bandwith.go

```diff
@@ -0,0 +1,77 @@
+package packetfabric
+
+import (
+	"errors"
+	"fmt"
+)
+
+const FlexBandwidthURI = "/v2/flex-bandwidth"
+
+// This struct represents a Flex Bandwidth
+// https://docs.packetfabric.com/api/v2/swagger/#/Flex%20Bandwidth/create_flex_bandwidth
+type FlexBandwidth struct {
+	Description      string `json:"description"`
+	AccountUUID      string `json:"account_uuid"`
+	SubscriptionTerm int    `json:"subscription_term"`
+	Capacity         string `json:"capacity"`
+	PoNumber         string `json:"po_number,omitempty"`
```

Comment:
As it is an optional field, it may be reasonable to change it to pointer type (i.e. `*string`) to be able to set it to `nil`.

Reply:
I don't think we have done that in any other resources (using pointer type when optional). All optional fields have the option `,omitempty` which omits them from the payload when not specified by the user.